lib/ex_subtil_backend/workflow/workflow_step.ex

```diff
@@ -0,0 +1,51 @@
+defmodule ExSubtilBackend.WorkflowStep do
+  @moduledoc """
+  The Workflow Step context.
+  """
+
+  require Logger
+
+  alias ExSubtilBackend.Workflows.Workflow
+  alias ExSubtilBackend.Jobs
+  alias ExSubtilBackend.Amqp.{JobFtpEmitter, JobGpacEmitter}
+
+  def start_next_step(%Workflow{id: workflow_id} = workflow) do
+    reference =
+      workflow.reference
+      |> String.to_integer
```

Comment:
⚠️ The `video.legacy_id` used has a `workflow.reference` is often prefixed by `NI_`, so the `String.to_integer` may crash!